src/components/repertoire/repertoire-card.tsx

```diff
@@ -14,9 +16,11 @@ const RepertoireCard = ({ song }: RepertoireCardProps) => {
     return false
```

Comment:
Don't need to change this as this isn't your PR code but I believe this could be written as 

```ts
return song.song_instruments.some((item) => item.division.length === 0)
```